src/pages/ProductDetail/BarChart.js

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect, useRef } from 'react';
+import styled from 'styled-components';
+import Chart from 'chart.js/auto';
+
+const StyledCanvas = styled.canvas`
+  width: ${props => props.width};
+  height: ${props => props.height};
```

Comment:
```suggestion
  width: ${({width}) => width};
  height: ${({height}) => height};
```
참고참고~~~
이렇게도 구조분해할당 할 수 있습니다.